java/code/src/com/suse/manager/model/attestation/AttestationFactory.java

```diff
@@ -239,4 +255,34 @@ public List<ServerCoCoAttestationReport> listCoCoAttestationReports(Server serve
                 .setFirstResult(offsetIn)
                 .list();
     }
+
+    /**
+     * Return a list of reports for a given server with filters
+     * @param user the user
+     * @param pc page control object
+     * @return returns a list or reports
+     */
+    public List<ServerCoCoAttestationReport> listCoCoAttestationReports(User user, PageControl pc) {
```

Comment:
I thought we have this method already. But this one is listing for a User, while the other is listing for a Server.
Maybe we should encode this in the names?
listCoCoAttestationReportsByUser() and listCoCoAttestationReportsByServer() ?

Reply:
Yes, good point. I will change the naming :+1: